yb-voyager/src/datafile/csvdatafile.go

```diff
@@ -30,29 +31,31 @@ func (df *CsvDataFile) SkipLines(numLines int64) error {
 }
 
 func (df *CsvDataFile) NextLine() (string, error) {
-	/*
-		// TODO: resolve the issue in counting bytes with csvreader
-		// here csv reader can be more useful in case filter table's columns
-		// currently using normal file reader will also work
+	var bytes bytes.Buffer
```

Comment:
Can we reuse the same bytes.Buffer (maybe after cleaning/resetting) in each iteration?

Reply:
yes we are already reusing it

Reply:
For each "record" a new Buffer will be created. We should consider making it a field of CsvDataFile.

Reply:
ok